src/main/java/com/liferay/faces/util/i18n/I18nBundleBase.java

```diff
@@ -171,13 +171,14 @@ public I18n getWrapped() {
 
 	/**
 	 * Returns a new message cache to be used by {@link I18nBundleBase}. The default implementation returns a Cache
-	 * instance obtained from {@link CacheFactory#getConcurrentCacheInstance(javax.faces.context.ExternalContext)}. This
-	 * method is called from the constructor of I18nBundleBase, so this method must not cause side effects and should
-	 * not expect I18nBundleBase (or its subclass) to be fully initialized.
+	 * instance obtained from {@link CacheFactory#getConcurrentCacheInstance(javax.faces.context.ExternalContext, int)}
+	 * (passing 16 as the initial cache capacity). The cache will be stored in the application map. This method is
+	 * called from the constructor of I18nBundleBase, so this method must not cause side effects and should not expect
+	 * I18nBundleBase (or its subclass) to be fully initialized.
 	 *
 	 * @param  externalContext  The external context associated with the current {@link FacesContext}.
 	 */
 	protected Cache<String, String> newConcurrentMessageCache(ExternalContext externalContext) {
```

Comment:
@stiemannkj1 Rather than passing a hard-coded 16, since `externalContext` is passed, then maybe this method should consult `WebConfigParam.I18nInitialCacheCapacity`? Either that, or change the method signature to be `newConcurrentMessageCache(ExternalContext externalContext, int initialCapacity)` and pass in the result from `WebConfigParam.I18nInitialCacheCapacity.getIntegerValue()` from the constructor?